v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
We use this one for targetting v4.5, 4.5.1 and 4.5.2 today.

I'd like this to be separated/copied into folders for each profile in the end so we can expose the APIs like they were intended to.